react/src/components/dropdown/SprkDropdown.stories.js

```diff
@@ -87,7 +95,7 @@ export const informational = () => (
             infoLine2: 'More Information',
           },
           value: 'choice-title-2',
-          isActive: false,
+          isDefault: true,
```

Comment:
Is dropdown breaking change? Forgot why we didnt add legacy stories. Do we not need them? @bobwise